pyspeckit/spectrum/readers/gbt.py

```diff
@@ -129,7 +156,7 @@ def read_gbt_target(sdfitsfile, objectname, verbose=False):
         for nod in nods:
             whnod = bintable.data['PROCSEQN'] == nod
             for onoff in ('ON','OFF'):
-                calOK = (calON - (onoff=='OFF'))
+                calOK = np.bitwise_xor(calON, onoff=='OFF')
```

Comment:
I wonder if this was actually supposed to be `calOK = calON & not (onoff == 'OFF')` ?  I don't see why `xor` logic should be applied here... even though that is a correct interpretation of `-` in this case.